reference/media-types.md

```diff
@@ -414,153 +450,53 @@ will render them as an array of Strings.
-  end
-
-  class Collection < Praxis::MediaTypeCollection
-    member_type Post
+  # ...  
 
+  class CollectionSummary < Praxis::Blueprint
     attributes do
+      attribute :name, String
+      attribute :size, Integer
       attribute :href, String
-      attribute :count, Integer
-      attribute :authors, Attributor::Collection.of(String)
     end
 
     view :link do
+      attribute :name
+      attribute :size
       attribute :href
-    end
-
-    view :aggregate do
-      attribute :href
-      attribute :count
-      attribute :authors
+      end
     end
   end
 end
 {% endhighlight %}
 
-This Post media type has some attributes you'd expect on a blog post, and its
-default view renders all of them. There is also a ```Post::Collection``` media
-type.  The collection has its own attributes that refer to properties of the
-collection itself rather than properties of the collection's members.
-
-This new collection media type is a first class media type like any other,
-except collection media types reference the type of members that the collection
-contains.  In this case, the ```Post::Collection``` media type has ```Post```
-as its member_type.  Like any other media_type, rendering a
-```MediaTypeCollection``` will use the views you have defined in it. 
-
-It is often the case that a media_type collection just wants to render itself
-as a simple array wrapping its containing members. For this reason, Praxis
-adds an additional DSL method to media_type collections called `member_view`. 
-Here is an example of how to use it:
-
-{% highlight ruby %}
-member_view :members, using: :default
-{% endhighlight %}
-
-Adding the above statement to our ```Post::Collection``` media_type 
-will create a new view called `:members`. Rendering this generates an array 
-containing the existing members rendered using their `:default` view:
-
-{% highlight javascript %}
-[
-  { id: 1, title: 'Title1', content: 'This is some text' },
-  { id: 2, title: 'Title2', content: 'And some more' },
-  { id: 3, title: 'Title3', content: 'Lorem ipsum' }
-]
-{% endhighlight %}
-
-
-Defining collections as inner classes within the related member_type has the
-advantage that can be used from other media types. Below are some examples of a
-`Blog` resource that refers to the `Post` collection in various ways.
-
-#### Embedding full collection contents
-
-It is possible to directly embed every `Post` into its `Blog`. In this example,
-when Praxis renders the view `default` for a `Blog`, it locates the
-`Collection` media type you defined for `Post`, and looks for a default view.
-If there is a default view directly on the collection, Praxis renders it. Since
-there isn't one, Praxis will iterate the members of the collection, each one a
-`Post` media type, and render them all using the default view for `Post`.
-
-Either `Post::Collection` or `Post` must have a `default` view in this case.
-
+To then use that from another media type, you would do the following:
```

Comment:
Would change to:
Then you can use that from the `Blog`s media type in the following way: